app/code/core/Mage/Customer/Model/Customer.php

```diff
@@ -73,6 +73,11 @@ class Mage_Customer_Model_Customer extends Mage_Core_Model_Abstract
      */
     const MINIMUM_PASSWORD_LENGTH = 6;
 
+    /**
+     * Maximum Password Length
+     */
+    const MAXIMUM_PASSWORD_LENGTH = 256;
```

Comment:
Never used

Reply:
It should be - it's used here: https://github.com/OpenMage/magento-mirror/blob/a5ad2ee47599400ef0066562315b300c7f581938/app/code/core/Mage/Customer/Model/Customer.php

@spinsch Where there any issues applying the patch? Or is this an issue with the patch released by Magento?

Reply:
Sorry. It was my fault - the release from Magento is ok.